articles/teams.md

```diff
@@ -20,7 +20,7 @@ Fleet's best practice teams:
 - `Compliance exclusions`: All contributors' test work computers or virtual machines (VMs). Used for validating workflows for Fleet customers or reproducing bugs in the Fleet product.
 - `📱🏢 Company-owned iPhones`: iPhones purchased by the organization that enroll to Fleet automatically via Apple Business Manager. For example, iPhones used by iOS Engineers.
 - `🔳🏢 Company-owned iPads`: iPads purchased by the organization that enroll to Fleet automatically via Apple Business Manager. For example, conference-room iPads.
-<!--- - `📱👔 BYOD iPhones`: End users' personal iPhones that have access to company resources. For example, Slack and Gmail. contributors' test iOS hosts. (BYOD iPhone features are coming soon) -->
+- `📱🔐 Personally-owned iPhones`: End users' personal iPhones that have access to company resources. For example, Slack and Gmail.
```

Comment:
```suggestion
- `📱🔐 Personally-owned (BYOD) iPhones`: End users' personal iPhones that have access to company resources. For example, Slack and Gmail.
```

I know you're not a fan of this but "BYOD" is how we, and many customers, refer to these iPhones. Can we throw BYOD in parenthesis here so that we can coach people in the "personally-owned" direction. "Personally-owned" means "BYOD."

Later, we might be able to get rid of "BYOD"

Reply:
Perhaps this as an edit? 
> End users' personal iPhones, like those enrolled through a BYOD program, that have access to company resources.

Let's discuss in the next dogfood meeting after the holiday break. From an admin and InfoSec perspective, we care about devices falling into two categories because that determines how we are going to manage them and what access they have -- owned by the company or owned by the employee. BYOD is a program or deployment type, not a device type. Using the term BYOD also does not align with Apple's current messaging. Apple refers to these devices as User-owned devices. I need to do a little more research into how Microsoft and Android refer to these device types. It also seems awkward to refer to these as "Personally-owned Bring Your Own Device iPhones". 

BYOD is not limited to iPhones -- computers (like those used by contractors), tablets/iPads, watches, Vision Pro, etc. can all be user-enrolled.

I also have this issue being tracked where we should consider dropping the word iPhones (and iPads) from the team names entirely: fleetdm/confidential#9192